utils/constants.ts

```diff
@@ -82,6 +84,8 @@ export const TOKEN_SYMBOLS_MAP = {
       [CHAIN_IDs.ARBITRUM_GOERLI]: "0xe39Ab88f8A4777030A534146A9Ca3B52bd5D43A3",
       [CHAIN_IDs.GOERLI]: "0xB4FBF271143F4FBf7B91A5ded31805e42b2208d6",
       [CHAIN_IDs.SEPOLIA]: "0x7b79995e5f793A07Bc00c21412e50Ecae098E7f9",
+      [CHAIN_IDs.ZK_SYNC]: "0x5AEa5775959fBC2557Cc8789bC1bf90A239D9a91",
```

Comment:
Should this be the native eth or weth address?

Reply:
Based on the other addresses (for Arbitrum and Optimism), these should be WETH addresses.